README.md

```diff
@@ -106,8 +119,9 @@ Below is an example configuration for a Lambda function that is part of a step f
       "Type": "Task",
       "Resource": "arn:aws:lambda:us-west-2:ACCOUNT:function:FUNCTION_NAME",
       "Parameters": {
-          "Changed": "parameters",
-          "_lumigo.$": "$._lumigo"
+          "size.$": "$.product.details.size",
+          "exists.$": "$.product.availability",
+          "_lumigo.$": "$.['_lumigo', 'null']"
```

Comment:
Why did you make these changes?

Reply:
Thought that's what you asked in [here](https://github.com/lumigo-io/lumigo-node/pull/501#discussion_r1720926189) - to have the readme match the step function's documentation on the [website](https://docs.lumigo.io/docs/step-functions).

Reply:
Following our discussion - reverted this change.